src/run.ts

```diff
@@ -84,14 +85,23 @@ function updateEnvFiles() {
 
 // run_fe_locally runs the frontend and its dependencies locally
 // @ts-ignore
-async function run_fe_locally(runner: LabeledProcessRunner, options: { stage: string }) {
-  const stage = options.stage;
-  await writeUiEnvFile(stage, true);
+async function run_fe_locally(runner: LabeledProcessRunner) {
+  const apiUrl = await getCloudFormationStackOutputValue(
+    "seds-jon-cdk",
+    "ApiUrl"
+  );
+
+  if (apiUrl) {
```

Comment:
Maybe I'm too bold here but I think we can assume this is there.

Reply:
I agree, but TypeScript disagrees.  Should we use `!` below to get around it?

Reply:
I like that!